# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/gtest/moz.build
# Commit: f6df4eb84932
# Full Hash: f6df4eb84932ddaccae4e53d4897234e0153913e
# Author: Chris Pearce <cpearce@mozilla.com>
# Date: 2019-06-14 10:00:22
# Regressor Bug: 1558364
# File Overlap Count: 1
# Description:
#   Bug 1558364 - Add MediaSpan and use it for TrackBuffersManager::mInputBuffer. r=jya
#   
#   As seen in this profile of a Twitch replay: https://perfht.ml/2K9Ydb3 we can
#   often end up spending time in TrackBuffersManager::CodedFrameProcessing()
#   shaving off bytes from the front off TrackBuffersManager::mInputBuffer. This
# ==============================================================================

diff -r e708fe457a95 -r f6df4eb84932 dom/media/gtest/moz.build
--- a/dom/media/gtest/moz.build	Fri Jun 14 00:37:25 2019 +0000
+++ b/dom/media/gtest/moz.build	Fri Jun 14 02:10:09 2019 +0000
@@ -34,6 +34,7 @@
     'TestMediaDataEncoder.cpp',
     'TestMediaEventSource.cpp',
     'TestMediaMIMETypes.cpp',
+    'TestMediaSpan.cpp',
     'TestMP3Demuxer.cpp',
     'TestMP4Demuxer.cpp',
     'TestOpusParser.cpp',